src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
```

Comment:
ball1.getRandomBall().toString() - toString() does not have to be called, since this method will be called not explicitly by default. 
This always happens when we pass an object to System.out.println, if the object does not Override toString(), then its link will be in the console.